src/pages/register.tsx

```diff
@@ -0,0 +1,122 @@
+import { Gender } from "enum/gender";
+import { Dayjs } from "dayjs";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { DatePicker } from "@mui/x-date-pickers/DatePicker";
+import CommonTextField from "@/components/public/CommonTextField";
+
+export default function Home() {
+  const [email, setEmail] = React.useState("");
+  const [password, setPassword] = React.useState("");
+  const [confirmPassword, setConfirmPassword] = React.useState("");
+  // const [isSubmit, setIsSubmit] = React.useState(false);
+  const [value, setValue] = React.useState<Dayjs | null>(null);
+  const [displayName, setDisplayName] = React.useState<string>("");
+  const [gender, setGender] = React.useState<string>("");
+
+  const handleDisplayNameChange = (
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void => {
+    setDisplayName(event.target.value);
+  };
+
+  function handleEmailChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setEmail(event.target.value);
+  }
+
+  function handlePasswordChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setPassword(event.target.value);
+  }
+
+  function handleConfirmPasswordChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setConfirmPassword(event.target.value);
+  }
+
+  function handleGenderChange(
+    event: SelectChangeEvent<string>
+  ): void {
+    setGender(event.target.value)
+  }
+
+  return (
+    <Stack spacing={3} alignItems="center" justifyContent="center" style={{ minHeight: "100vh" }}>
+      <Logo width={119} height={119} />
+
+      <Box style={{ width: "50vw" }}>
+        <CommonTextField
+          header="Username"
+          placeholder="Display Name"
+          value={displayName}
+          handleValueChange={handleDisplayNameChange}
+          char_limit={100}
+          isErr={false}
+          errMsg=""
+        />
+      </Box>
+
+      <Box style={{ width: "50vw", marginTop: 0 }}>
+        <CommonTextField
+          header="Email"
+          placeholder="Email"
+          value={email}
+          handleValueChange={handleEmailChange}
+          isErr={false}
+          errMsg=""
+        />
+      </Box>
+
+      <Box style={{ width: "50vw" }}>
+        <PasswordTextFeild
+          header="Password"
+          placeholder="Password"
+          value={password}
+          handleValueChange={handlePasswordChange}
+          isErr={false}
+          errMsg=""
+        />
+      </Box>
+
+      <Box style={{ width: "50vw" }}>
+        <PasswordTextFeild
+          header="Confirm Password"
+          placeholder="Confirm Password"
+          value={confirmPassword}
+          handleValueChange={handleConfirmPasswordChange}
+          isErr={false}
+          errMsg=""
+        />
+      </Box>
```

Comment:
I don't see any error handling. Add them please.

Reply:
When the "password" and "confirm password" don't go the same way, should both text boxes error?

Reply:
And why you need additional helper text which doesn't look different from "comfirm password" helper text.
Is it possible to include the error in "errMsg" attribute?

Reply:
Now both text boxes error when they don't go the same way and I already put helper text in "errMsg" attribute.